README.md

```diff
@@ -1,3 +1,11 @@
 # IBM WebSphere Liberty Operator
 
-The WebSphere Liberty Operator allows you to deploy and manage applications running on WebSphere Liberty into Kubernetes-based platforms, such as Red Hat OpenShift. You can also perform Day-2 operations such as gathering traces and dumps using the operator
+The WebSphere Liberty Operator allows you to deploy and manage applications running on WebSphere Liberty into Kubernetes-based platforms, such as Red Hat OpenShift. You can also perform Day-2 operations such as gathering traces and dumps using the operator.
+
+# Issues and Contributions
+
+For issues relating specifically to the operator, please use the [GitHub issue tracker](https://github.com/WASdev/websphere-liberty-operator/issues). For more general issue relating to IBM WebSphere Application Server Liberty you can [get help](https://developer.ibm.com/wasdev/help/) through the WASdev community or, if you have production licenses for WebSphere Application Server, via the usual support channels. We welcome contributions following [our guidelines](https://github.com/WASdev/wasdev.github.io/blob/master/CONTRIBUTING.md).
```

Comment:
Picky but you are missing an 's' on the end of 'issue'
Also the help link 'https://developer.ibm.com/wasdev/help/' takes you to a page which is just a redirect to:
https://openliberty.io/support/
Is it worth linking directly?